File: 03. Programming - C#/TestPreparation3Example/Person.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TestPreparation3Example
{
   abstract  class Person
    {
        private string name;
        public string Name
        {
            get
            {
                return name;
            }
            set
            {
                name = value;
            }
        }

        private string egn;
        public string Egn
        {
            get
            {
                return egn;
            }
            set
            {
                egn = value;
            }
        }

        private string gender;
        public string Gender
        {
            get
            {
                return gender;
            }
            set
            {
                gender = value;
            }
        }

        public Person(string name, string egn,string gender)
        {
            this.name = name;
            this.egn = egn;
            this.gender = gender;
        }
    }
}

File: 02. Programming - Java/Lesson9/src/uni/fmi/bachelors/Person.java
package uni.fmi.bachelors;

public abstract class Person {

	private String name;
	private String gender;
	private String EGN;
	
	public Person(String name, String gender, String EGN) {
		this.name=name;
		this.gender=gender;
		this.EGN=EGN;
		
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getGender() {
		return gender;
	}

	public void setGender(String gender) {
		this.gender = gender;
	}

	public String getEGN() {
		return EGN;
	}

	public void setEGN(String eGN) {
		EGN = eGN;
	}
	
	

}
